packages/design-system/src/types/Table/Table.d.ts

```diff
@@ -30,8 +20,23 @@ export interface TableProps {
    * This prop will only be used when the `Table` `scrollable` prop is set and the table width is wider than the viewport.
    */
   scrollableNotice?: React.ReactNode;
+  /**
+   * A stackable variation of the table.
+   * When `stackable` is set, `id` or `headers` prop is required in `Table`
+   */
+  stackable?: boolean;
+  /**
+   * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
+   */
+  stackableBreakpoint?: TableStackableBreakpoint;
+  /**
+   * A striped variation of the table.
+   */
+  striped?: boolean;
```

Comment:
if we add a `striped` prop we should do the same for `borderless`

Reply:
The zebra `striped` prop is applicable for both the border and borderless (via className='ds-c-table--borderless') tables. I don't think we need another setting for borderless then.

Reply:
Ah yes, for stacked table, the layout is bordered, we haven't checked the borderless class to style it accordingly.

Reply:
Fixed it to have the borderless layout for stacked table for className='ds-c-table--borderless'